common/db/migrations/145_update_agent_rlshps.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'utils'
+
+agent_relationships = %w[agent_relationship_associative_relator
+                         agent_relationship_earlierlater_relator agent_relationship_parentchild_relator agent_relationship_subordinatesuperior_relator
```

Comment:
Maybe these should be on separate lines? Like:
agent_relationship_associative_relator
agent_relationship_earlierlater_relator 
agent_relationship_parentchild_relator 
agent_relationship_subordinatesuperior_relator
...